index.rst

```diff
@@ -10,20 +10,849 @@
+- The database must be available and rapidly synced to at all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
+- Summit tooling, including the Scheduler, must have immediate access to the database (`FAFF-REQ-0056`_).
+
+..
+   Plot Visualization
+   ^^^^^^^^^^^^^^^^^^^
+   Use and expansion of the plot visualization tool.
+   Also explain the current use of RubinTV
+
+.. _Deliverable 4:
+
+Deliverable 4: Required Non-Scalar Metrics
+------------------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+  4. (`SITCOM-180`_) Provide a list of required non-scalar metrics are required and cannot be computed with analysis_tools.
+     Suggest a mechanism (work flow) to perform the measurement, document the finding, evaluate any trend (if applicable), then present it to the stakeholders.
+
+
+.. related to https://confluence.lsstcorp.org/display/LSSTCOM/Displaying+scalar+fields+as+a+function+of+other+parameters
+
+This charge question covers the issue of calculating and displaying the trending of scalar fields.
+Scalar fields are single value metrics, but calculated per spatial element, as described in `Deliverable 3`_.
+This charge question deals with adding a third dimension to the scalar field, then calculating and displaying this data to the user.
+For example, this could be displaying the PSF width for each detector as a function of elevation, or sky transparency as a function of time.
+As discussed above, both of these examples deal with aggregated (binned) data.
+
+Currently, `analysis_tools`_ computes a bundle of single-valued (scalar) metrics on individual visits.
+With small modifications, the package could persist arrays of metric values (e.g., per detector or finer granularity) that could be aggregated and visualized in flexible ways by downstream tooling.
+The package already produces and persists static plots for displaying scalar fields in focal plane coordinates.
+
+After analyzing the use-cases, including hypotheticals not detailed in the report, it was decided that there is not a use-case where we are unable represent a scalar field with respect to a third axis (e.g. time, elevation etc) as a single valued metric (e.g. a mean, or standard deviation), so long as the desired aggregation is supported.
+Taking the examples discussed above, one would reduce the scalar field to a number of scalar metrics, such as the mean PSF width, or the standard deviation about that mean, as a function of elevation.
+Similarly, the sky transparency could be handled by looking at the standard deviation compared to a 2-d map of a photometric night.
+Reducing a scalar field to a scalar metric creates a more generalizable framework to communicate data, however, it comes a the expense of removing information.
```

Comment:
"it comes a the expense" -> "it comes at the expense"

"Reducing a scalar field to a scalar metric creates a more generalizable framework to communicate data"

Not sure I agree with this statement. Isn't a "scalar field" more general than a "scalar metric"? You can always reduce a "scalar field" into a "scalar metric" but you can't do the reverse.

Reply:
typo fixed.

I'm confused... yes, going from a scalar field to a metric results in a loss of information, so you can't go the other way. That's what the text says, no?